### DISCUSSION TITLE
Is custom `template.tex` partial not considered?

### DISCUSSION THREAD

**Discussion by iusgit** [quarto-dev/quarto-cli#3522] [Q&A]
Hi,

I have changed partial `template.tex` and added to the partials list:
```
template-partials:
        - "resources/formats/pdf/partials/title.tex"
        - "resources/formats/pdf/partials/before-body.tex"
        - "resources/formats/pdf/partials/biblio.tex"
        - "resources/formats/pdf/partials/template.tex"
```

However, it is not read. 

Is that by design or – consequently – am I doing something wrong?

**Comment by dragonstyle** [comment]
If you're replacing the core template, you should just use the `template` option to provide your template directly. I will improve our documentation to make this more clear- currently definitely seems like you should be able to replace it as a partial...

**Reply by iusgit** [reply]
Thanks, that works!
